lib/model.js

```diff
@@ -1180,10 +1180,12 @@ Model.prototype.scope = function(option) {
         if (Array.isArray(option.method) && !!self.options.scopes[option.method[0]]) {
           scopeName = option.method[0];
           scope = self.options.scopes[scopeName].apply(self, option.method.splice(1));
+          conformOptions(scope, self);
```

Comment:
Hmm, this code could be simplified a bit to avoid too much duplication. If we don't call the scope function here, but just set scope, (and possibly arguments), and then call `isFunction` and `conformOptions` after the else clause we only need the code once :)